srcpkgs/ldc/template

```diff
@@ -1,24 +1,25 @@
 # Template file for 'ldc'
 pkgname=ldc
-version=1.13.0
-revision=2
+version=1.19.0
+revision=1
+archs="*-glibc"
```

Comment:
this is not a real fix, if the pkg was built for musl before you should make it work again.

Reply:
It's not entirely clear why it doesn't work, and I don't have a musl system to test.  There is an upstream issue tracking it.

Reply:
`-glibc` doesn't even exists. You want `~*-musl`